src/main/java/io/odpf/firehose/metrics/Metrics.java

```diff
@@ -60,4 +74,18 @@
     public static final String ERROR_MESSAGE_CLASS_TAG = "class";
     public static final String NON_FATAL_ERROR = "nonfatal";
     public static final String FATAL_ERROR = "fatal";
+
+    //SINK OBJECT STORAGE TAG
+    public static final String SINK_OBJECT_STORAGE_SCOPE_TAG = "scope";
```

Comment:
if we keep these constants in a seperate file we don't have to add the prefix of SINK_OBJECT_STORAGE it is not readable.

Reply:
will remove any `SINK_OBJECT_STORAGE` constant to another metric